app/utils/large_files.py

```diff
@@ -0,0 +1,35 @@
+import os
+from dotenv import load_dotenv
+import langchain
+from langchain.embeddings.openai import OpenAIEmbeddings
+from langchain.text_splitter import CharacterTextSplitter
+from langchain.vectorstores import FAISS 
+from langchain.chains.question_answering import load_qa_chain
+from langchain.llms import OpenAI
+from langchain import PromptTemplate 
+
+load_dotenv()
+
+langchain.embeddings.openai.api_key = os.getenv("OPENAI_API_KEY")
+
+def useEmbeddings(text: str,question:str):
+    # Splitting up the text into smaller chunks for indexing
+    text_splitter = CharacterTextSplitter(        
+        separator = "\n",
+        chunk_size = 1000,
+        chunk_overlap  = 100, #striding over the text
+        length_function = len,
+    )
+    texts = text_splitter.split_text(text)
+    #create and store embeddings for each chunck 
```

Comment:
where are stored the embeddings?

Reply:
docsearch = FAISS.from_texts(texts, embeddings)
stored in docsearch , faiss utilize OpenAiEmbeddings to create vector representations for texts

Reply:
ok so it's not stored in a DB. just into a variable